pkgs/by-name/fo/forgejo/generic.nix

```diff
@@ -128,6 +128,10 @@ buildGoModule rec {
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
+  preInstall = ''
+    mv "$GOPATH/bin/forgejo.org" "$GOPATH/bin/gitea"
```

Comment:
Any reason not to symlink gitea->forgejo.org? It would be nice to move away from gitea things in this package eventually.

Reply:
Upstream still uses `gitea` as binary name, as per https://codeberg.org/forgejo/forgejo/src/commit/9ebdc0993980ceddda103fa2e6dd82f18cd52f77/Makefile#L63

In my out-of-tree variant I nonetheless opted to use `forgejo` as well and have `gitea` as a symlink.
But I wouldn't do the same in nixpkgs just yet.

But I don't think we would want `forgejo.org` as binary name either way, because this is not what the upstream docs reference and really just an implementation detail of our `go/module.nix` in nixpkgs, if that makes sense